gnd/src/main/java/com/google/android/gnd/ui/map/gms/GoogleMapsMapAdapter.java

```diff
@@ -277,28 +295,45 @@ private void addMapPin(MapPin mapPin) {
   }
 
   private void addMapPolyline(MapPolygon mapPolygon) {
-    for (ImmutableSet<Point> vertices : mapPolygon.getVertices()) {
-      PolylineOptions options = new PolylineOptions();
-
-      // Read-only
-      options.clickable(false);
-
-      // Add vertices to PolylineOptions
-      stream(vertices).map(GoogleMapsMapAdapter::toLatLng).forEach(options::add);
+    PolylineOptions options = new PolylineOptions();
+    // Read-only
+    options.clickable(false);
+    // Add vertices to PolylineOptions
```

Comment:
This comment is in the wrong place; in this step we're converting the vertices to a list of LatLng (no comment needed unless it's clarifying what's already clear in code).

Reply:
updated.